src/@chakra-ui/stories/Colors.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import { type ReactNode } from "react"
+import capitalize from "lodash/capitalize"
+import { Box, Flex, HStack, Square, Stack, Text } from "@chakra-ui/react"
+import type { Meta, StoryObj } from "@storybook/react"
+
+import colors from "@/@chakra-ui/foundations/colors"
+import semanticTokens from "@/@chakra-ui/semanticTokens"
+
+const semanticTokenColors = semanticTokens["colors"]
+
+import Heading from "@/components/Heading"
+
+const meta = {
+  title: "Design System/Colors",
+  parameters: {
+    // Do not create snapshots for any stories in the file.
+    chromatic: { disableSnapshot: true },
+  },
+} satisfies Meta
+
+export default meta
+
+const primitiveColorKeys = ["gray", "blue", "orange"]
+export const Primitives: StoryObj = {
+  render: () => {
+    const primitiveColorsMap = Object.entries(colors)
+      .filter((obj) => primitiveColorKeys.includes(obj[0]))
+      .reduce<{ [tokenKey: string]: [string, string][] }>(
+        (acc, [key, value]) => {
+          const tokenMap = Object.entries(value)
+          return {
+            ...acc,
+            [key]: tokenMap,
+          }
+        },
+        {}
+      )
+
+    return (
+      <Stack direction="column" gap="16">
+        {primitiveColorKeys.map((color) => (
+          <ColorGroupWrapper key={color} color={color}>
+            <HStack>
+              {primitiveColorsMap[color].map(([tokenKey, value]) => (
+                <Stack key={`${color}${tokenKey}`} direction="column">
+                  <Box>
+                    <Square size="20" bg={`${color}.${tokenKey}`}></Square>
+                  </Box>
+                  <Stack direction="column">
+                    <Text size="sm">{value}</Text>
+                    <Text size="sm">
+                      {color}.{tokenKey}
```

Comment:
Good job @TylerAPfledderer. I like how we are displaying the colors in the `Primitives` case.

Two things I think would be useful for devs to display on the semantic tokens story:
1. code example or semantic token usage
2. primitive color reference

For example, for the primary "base" token we could add:
1. how it would be used in code: `primary.base`
2. the primitive color used by this token: `blue.500` or `orange.500`

Reply:
@pettinarip will do! I think I have an approach in mind.

Reply:
@pettinarip I updated the labelling of the semantic tokens. It got a little tricky trying to come up with a good way to extract the foundation tokens for each. I don't want to try and spend too much time trying to come up with some generator that would get messy in this PR.

Reply:
Sure. Cool, it looks more useful now with the actual tokens that we use on the code, thanks!